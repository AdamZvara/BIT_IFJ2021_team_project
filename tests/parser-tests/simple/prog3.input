require "ifj21"

global f : function(integer)

function f(x : integer)
	local a : integer = x + 2
	write(a," ", x,"\n")
	if a ~= x + 2 then
		write("NOT EQUAL\n")
	else write("EQUAL\n")
	end
	if a == x + 2 then
		write("EQUAL\n")
		local a : number = 2.21
		write(a,"\n")
	else end
	if a > x then else end
end

f(2)
