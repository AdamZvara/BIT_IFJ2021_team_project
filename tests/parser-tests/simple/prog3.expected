4 2
EQUAL
EQUAL
0x1.1ae147ae147aep+1
